js/personal.js

```diff
@@ -76,23 +96,37 @@ View.prototype = {
     },
     renderContent: function () {
         this._persons.sortBySize();
-
-        var html = Handlebars.templates['personal']({
+        var context = {
             loaded: this._persons.isLoaded(),
             persons: this._persons.getAll(),
             appName: t('facerecognition', 'Face Recognition'),
             welcomeHint: t('facerecognition', 'Here you can see photos of your friends that are recognized'),
             loadingMsg: t('facerecognition', 'Looking for your recognized friends'),
             loadingIcon: OC.imagePath('core', 'loading.gif'),
             emptyMsg: t('facerecognition', 'Your friends have not been recognized yet'),
-            emptyHint: t('facerecognition', 'Please, be patient'),
-        });
+            emptyHint: t('facerecognition', 'Please, be patient')
+        };
+
+        if (this._persons.getActive() !== undefined)
+            context.person = this._persons.getActive();
+
+        var html = Handlebars.templates['personal'](context);
         $('#div-content').html(html);
 
         const observer = lozad('.face-preview');
         observer.observe();
 
         var self = this;
+
+        $('#facerecognition .person-name').click(function () {
+            var id = $(this).parent().data('id');
+            self._persons.loadPerson(id).done(function () {
+                self.renderContent();
+            }).fail(function () {
+                alert('D\'Oh!. Could not load person..');
```

Comment:
is this alert, like those JS alert() functions pop-up message boxes?:)

Reply:
I have to replace them with notifications and better messages ... Although theoretically it should never be executed ..